spread.yaml

```diff
@@ -522,4 +522,44 @@ suites:
             distro_purge_package qemu genisoimage sshpass
             snap remove ubuntu-image
 
+    tests/performance/:
+        summary: Performance tests for snapd
+        manual: true
+        environment:
+            RES_FILE: "$PROJECT_PATH/test.perf"
+        prepare: |
+            . "$TESTSLIB/prepare.sh"
+            if [[ "$SPREAD_SYSTEM" == ubuntu-core-16-* ]]; then
```

Comment:
I'm somewhat worried by the number of copy-pasted prepare/restore code we have in all the suites.

Reply:
yeah, probably not for this PR but we should think what we can do to avoid the amount of copy/paste

Reply:
Perhaps we should apply the aggregation of prepare/restore logic like what I did a while ago for some of the other tasks? The original PR was closed but I think the idea is sound.